__tests__/__theia__/extension.theia.test.ts

```diff
@@ -9,51 +9,266 @@
+        const favoriteProfile = await driverChrome.getFavoritePrfileNameFromDatasets();
+        expect(favoriteProfile).to.equal("TestSeleniumProfile");
+    });
+
+    it("Should Add Profile to Favorites under USS", async () => {
+        await driverChrome.clickOnDatasetsTab();
+        await sleep(SleepForTwoSec);
+        await driverChrome.clickOnUssTabs();
+        await sleep(SleepForTwoSec);
+        await driverChrome.addProfileToFavoritesInUss();
+        await driverChrome.clickOnFavoriteTabInUss();
+        await sleep(SleepForTwoSec);
+        const favoriteProfile = await driverChrome.getFavoritePrfileNameFromUss();
+        expect(favoriteProfile).to.equal("TestSeleniumProfile");
+    });
+
+    it("Should Add Profile to Favorites under JOBS", async () => {
+        await driverChrome.clickOnUssTabs();
+        await sleep(SleepForTwoSec);
+        await driverChrome.clickOnJobsTab();
+        await driverChrome.addProfileToFavoritesInJobs();
+        await sleep(SleepForTwoSec);
+        await driverChrome.clickOnFavoriteTabInJobs();
+        await sleep(SleepForTwoSec);
+        const favoriteProfile = await driverChrome.getFavoritePrfileNameFromJobs();
+        expect(favoriteProfile).to.equal("TestSeleniumProfile");
+    });
+
+    after(async () => driverChrome.closeBrowser());
+});
+
+describe("Remove Profile from Favorites", () => {
+    before(async () => {
+        await driverChrome.openBrowser();
+        await sleep(SleepForTwoSec);
+        await driverChrome.OpenTheiaInChrome();
+        await sleep(SLEEPTIME);
+        await driverChrome.clickOnZoweExplorer();
+    });
+
+    it("Should Remove Profile from Favorites under DATA SETS", async () => {
+        await driverChrome.clickOnFavoriteTabInDatasets();
+        await sleep(SleepForTwoSec);
+        await driverChrome.clickOnFavoriteProfileInDatasets();
+        await driverChrome.removeFavoriteProfileFromDatasets();
+        await sleep(SleepForTwoSec);
+        await driverChrome.refreshBrowser();
+        await sleep(SLEEPTIME);
+        const favoriteProfile = await driverChrome.verifyRemovedFavoriteProfileInDatasets();
+        expect(favoriteProfile).to.equal(true);
+    });
+
+    it("Should Remove Profile from Favorites under USS", async () => {
+        await driverChrome.clickOnDatasetsTab();
+        await sleep(SleepForTwoSec);
+        await driverChrome.clickOnUssTabs();
+        await sleep(SleepForTwoSec);
+        await driverChrome.clickOnFavoriteTabInUss();
+        await driverChrome.clickOnFavoriteProfileInUss();
+        await driverChrome.removeFavoriteProfileFromUss();
+        await sleep(SleepForTwoSec);
+        await driverChrome.refreshBrowser();
+        await sleep(SLEEPTIME);
+        const favoriteProfile = await driverChrome.verifyRemovedFavoriteProfileInUss();
+        expect(favoriteProfile).to.equal(true);
+    });
+
+    // it("Should Remove Profile from Favorites under JOBS", async () => {
```

Comment:
Can we remove it?

Reply:
do you mean Line number 203 (commented code) should remove?

Reply:
Yep! :)

Reply:
Removed commented block